solutions/usaco-1230.mdx

```diff
@@ -18,75 +27,176 @@ author: Aditya Gupta
 
 ```cpp
 #include <bits/stdc++.h>
-
 using namespace std;
 
-typedef long long ll;
+// Cow i wants to visit cow a[i] and gets v[i] points
+vector<int> a, v;
+// Reversed_graph[i] stores the cows that want to go to farm i
```

Comment:
```suggestion
// reversed_graph[i] stores the cows that want to go to farm i
```
i think its better if you lowercase the first letter for other comments because thats what every other solution uses